drift-transport-netty/src/main/java/io/airlift/drift/transport/netty/HeaderTransport.java

```diff
@@ -271,6 +313,53 @@ public boolean isSupportOutOfOrderResponse()
         {
             return supportOutOfOrderResponse;
         }
+
+        @Override
+        public int refCnt()
+        {
+            return message.refCnt();
+        }
+
+        @Override
+        public ReferenceCounted retain()
+        {
+            message.retain();
+            return this;
+        }
+
+        @Override
+        public ReferenceCounted retain(int increment)
+        {
+            message.retain(increment);
+            return this;
+        }
+
+        @Override
+        public ReferenceCounted touch()
+        {
+            message.touch();
+            return this;
+        }
+
+        @Override
+        public ReferenceCounted touch(Object hint)
+        {
+            message.touch(hint);
+            return this;
+        }
+
+        @CheckReturnValue(when = UNKNOWN)
```

Comment:
Did you mean to add this here and in other places?

Reply:
Spot bugs reports this as an error when called unless you check the response code.  In this case, I don't care about which release call actually frees the data, so I disabled it  Here is the actual message:
```
Return value of io.airlift.drift.transport.netty.TChannelBufferOutputTransport.release() ignored, but method has no side effect [io.airlift.drift.transport.netty.server.ThriftServerHandler] At ThriftServerHandler.java:[line 249] RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT
```